src/metatrain/experimental/soap_bpnn/tests/test_torchscript.py

```diff
@@ -21,12 +20,15 @@ def test_torchscript():
     model = SoapBpnn(MODEL_HYPERS, dataset_info)
     model = torch.jit.script(model)
 
-    system = ase.Atoms(
-        "OHCN",
-        positions=[[0.0, 0.0, 0.0], [0.0, 0.0, 1.0], [0.0, 0.0, 2.0], [0.0, 0.0, 3.0]],
+    system = System(
+        types=torch.tensor([6, 1, 8, 7]),
+        positions=torch.tensor(
+            [[0.0, 0.0, 0.0], [0.0, 0.0, 1.0], [0.0, 0.0, 2.0], [0.0, 0.0, 3.0]]
+        ),
+        cell=torch.zeros(3, 3),
```

Comment:
Here you completely avoid the `dtype` argument and it seems to work. Therefore, I think you can neglect this everywhere.

Reply:
I kept the dtypes that were already there. If I change these, others that are in different tests will still be there. Besides this, I'm not sure it would work everywhere. I think this is for a different PR